cmd/minikube/cmd/start.go

```diff
@@ -865,10 +865,10 @@ func validateFlags(cmd *cobra.Command, drvName string) {
 		}
 	}
 
-	// check that kubeadm extra args contain only whitelisted parameters
+	// check that kubeadm extra args contain only allowed parameters
 	for param := range config.ExtraOptions.AsMap().Get(bsutil.Kubeadm) {
-		if !config.ContainsParam(bsutil.KubeadmExtraArgsWhitelist[bsutil.KubeadmCmdParam], param) &&
-			!config.ContainsParam(bsutil.KubeadmExtraArgsWhitelist[bsutil.KubeadmConfigParam], param) {
+		if !config.ContainsParam(bsutil.KubeadmExtraArgsAllow[bsutil.KubeadmCmdParam], param) &&
```

Comment:
how about KubeadmExtraArgsAllowList

Reply:
I started with AllowList (even if this is a dictionary, rather than a list), but removed it in preference to following the idea that one should omit words that are obvious given a variable's type declaration or are clear from the surrounding context.